js/hubs/components/Release.js

```diff
@@ -200,10 +227,40 @@ const ReleaseComponent = ({ metadataSsr, releasePubkey, hubPubkey }) => {
                 hubPubkey={hubPubkey}
               />
               {userIsRecipient && (
-                <Royalty
-                  release={releaseState.tokenData[releasePubkey]}
-                  releasePubkey={releasePubkey}
-                />
+                <Box sx={{ textAlign: 'left' }}>
```

Comment:
close release behavior should only be visible to authority, not recipients. We can handle this when moving it into the settings modal.